public/config.json

```diff
@@ -6,6 +6,8 @@
     "analytics_server_url": "https://plausible.10bedicu.in",
     "static_header_logo": "https://cdn.coronasafe.network/header_logo.png",
     "static_light_logo": "https://cdn.coronasafe.network/light-logo.svg",
+    "static_ohc_light_logo": "https://cdn.coronasafe.network/ohc_logo_light.png",
+    "static_ohc_green_logo": "https://cdn.coronasafe.network/ohc_logo_green.png",
     "static_black_logo": "https://cdn.coronasafe.network/black-logo.svg",
     "static_dpg_white_logo": "https://digitalpublicgoods.net/wp-content/themes/dpga/images/logo-w.svg",
     "static_coronasafe_logo": "https://3451063158-files.gitbook.io/~/files/v0/b/gitbook-legacy-files/o/assets%2F-M233b0_JITp4nk0uAFp%2F-M2Dx6gKxOSU45cjfgNX%2F-M2DxFOkMmkPNn0I6U9P%2FCoronasafe-logo.png?alt=media&token=178cc96d-76d9-4e27-9efb-88f3186368e8",
```

Comment:
Shouldn't this be removed?
```suggestion
```

Reply:
I was thinking we could keep it there, in case it needs to be used later?

Reply:
Alright. However, make a PR for the changes made here to the deploy configs repo too.